src/gui/gui.cc

```diff
@@ -863,6 +863,8 @@ void PCSX::GUI::endFrame() {
                 }
                 ImGui::MenuItem(_("Show Registers"), nullptr, &m_registers.m_show);
                 ImGui::MenuItem(_("Show Assembly"), nullptr, &m_assembly.m_show);
+                if (PCSX::g_emulator->m_psxCpu->isDynarec())
+                    ImGui::MenuItem(_("Show DynaRec Disassembly"), nullptr, &m_disassembly.m_show);
```

Comment:
We probably want an else statement here to show the same string in a disabled text fashion.

Reply:
The goal here was to prevent it from appearing in the debug menu if not running on a dynarec CPU. Are you wanting this to still show the menu item but as disabled and greyed out?

Reply:
That's correct. Having a consistent UI is important. Not displaying a menu item under certain circumstances is bad UX because it confuses users when it happens. It's better to display the same entry, disabled, to convey to users they'd be able to use this under different settings. Maybe even add a hover text to explain why it's disabled and what to do to enable it.